src/store/getters.ts

```diff
@@ -606,18 +616,17 @@ export const getXproRequestTypeCd = (state: StateIF): XproNameType => {
 /** Get entity type options (short list only). */
 export const getEntityTypeOptions = (state: StateIF): Array<EntityI> => {
   const bcOptions: SelectOptionsI[] = getEntityTypesBC(state)?.filter(x => {
-    // Set shortlisted entity types for BC Move and Restoration requests.
-    if (
-      (isContinuationIn(state) || isRestoration(state)) &&
-      isLocationBC(state)
-    ) {
+    if (isContinuationIn(state) && isLocationBC(state)) {
+      // Shortlist order: Limited Company, ULC
+      if (x.value === EntityTypes.CUL) {
+        x.shortlist = true
+        x.rank = 2
+      }
```

Comment:
Limited Company (CR) is automatically added as the first element in the short list.